app/assets/stylesheets/pages/home/_ukm.scss

```diff
@@ -0,0 +1,106 @@
+.ukm{
+	margin-top: 33px;
+
+	&__title-wrap{
```

Comment:
remove title wrap, and add styling to the title instead

Reply:
already to deleted this class